DirectEmployers/MyJobs
# Reporting - dynamic report intentions [PD-2962]

## What's this PR do?
This PR makes the report intention select box dynamic.

As of right now, I'd like to ensure this meets the expectations of the team. I think it's going in the right direction but additional input is always nice.
## Screenshots (if appropriate)
![2017-02-28-132455_392x344_scrot](https://cloud.githubusercontent.com/assets/3379265/23469702/4f6d11ec-fe72-11e6-86d0-9a31920e4cda.png)
## Where should the reviewer start?
Take a look at the initial reporting screen. There should be more than two options.
## How should this be manually tested?
Click "PRM Reports". Note that you've got a bunch of fields to fill out.
Click one of the other options. Note that you get redirected to analytics.
## Any background context you want to provide?
